src/components/responsive-table/cell.tsx

```diff
@@ -33,15 +32,11 @@ const Cell = ({
   const icon =
     IS_FIRST_CELL && IS_HIDDEN_COLUMNS ? (
       <Icon
```

Comment:
maybe it should be ButtonIcon because it interactive element and a user should have the opportunity to choose this item via keyboard 

it's not required, but can be improved

Reply:
makes sense, I will add it in the next PR